cms/templates/js/signatory-editor.underscore

```diff
@@ -0,0 +1,33 @@
+     <div class="signatory-panel-default">
+
+       <% if (!is_editing_all_collections) { %>
+       <a class="signatory-panel-close" href="javascript:void(0);" data-tooltip="Close">
+          <i class="icon fa fa-close" aria-hidden="true"></i>
+          <span class="sr action-button-text"><%= gettext("Close") %></span>
+       </a>
+       <a class="signatory-panel-save"  href="javascript:void(0);" data-tooltip="Save">
+          <i class="icon fa fa-save" aria-hidden="true"></i>
+          <span class="sr action-button-text"><%= gettext("Save") %></span>
+       </a>
+       <% } else if (signatories_count > 1 && (total_saved_signatories > 1 || isNew) ) { %>
+       <a class="signatory-panel-delete" href="#" data-tooltip="Delete">
+          <i class="icon fa fa-trash-o" aria-hidden="true"></i>
+          <span class="sr action-button-text"><%= gettext("Delete") %></span>
+       </a>
+       <% } %>
+       <div class="signatory-panel-header">Signatory <%= signatory_number %></div>
+       <div class="signatory-panel-body">
+         <fieldset class="collection-fields signatory-fields">
+                <div class="input-wrap field text add-signatory-name">
+                    <label><%= gettext("Name ") %></label>
+                    <input class="collection-name-input input-text signatory-name-input" name="signatory-name" type="text" placeholder="<%= gettext("This is the Name of the Signatory") %>" value="<%= name %>">
+                    <span class="tip tip-stacked"><%= gettext("Max 30 characters") %></span>
+                </div>
+                <div class="input-wrap field text add-signatory-title">
+                    <label><%= gettext("Title ") %></label>
+                    <input class="collection-name-input input-text signatory-title-input" name="signatory-title" type="text" placeholder="<%= gettext("This is the Title of the Signatory") %>" value="<%= title %>">
+                    <span class="tip tip-stacked"><%= gettext("Max 30 characters") %></span>
```

Comment:
I love having this guidance in the UI!
I don't think it's enforced though, is this true? I am able to specify names and titles > 30 chars and save the certificate without causing an error.